### DISCUSSION TITLE
Esp8684 - anybody already played around with one?

### DISCUSSION THREAD

**Discussion by isarrider** [arendst/Tasmota#17686] [Q&A] [answered]
HI,

I have a ESPC2-12 dev board, which houses an ESP8684...
Already some experiences?

BR;
Alex

**Comment by Jason2866** [comment, marked as answer]
Since here is everything Tasmota related, no. Simple reason there is no Arduino support for the C2.
C2 needs IDF 5.0 to be supported. Arduino is based on IDF 4.4.3.
Afaik Arduino will use IDF 5.0 as base in the second half of the year. Don't hold your breath waiting to be supported in Tasmota.

**Reply by isarrider** [reply]
ah, awesome - so basically sit it out then...
Thank you!
